duckduckgo/macos-browser
# Set REVIEW Swift flag for review builds

Task/Issue URL: https://app.asana.com/0/1199230911884351/1203937201668191/f

**Description**:
Add missing flag for Review build configuration.

**Steps to test this PR**:
1. Grab the build from the Asana task. It has been created from this branch ([link to the workflow run](https://github.com/more-duckduckgo-org/macos-browser/actions/runs/4134761324))
2. Verify that Debug menu is available on the menu bar.



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
